Tests/TuistKitTests/Generator/TestData/GeneratedWorkspaceDescriptor+TestData.swift

```diff
@@ -0,0 +1,20 @@
+
+import Basic
+import Foundation
+import XcodeProj
+@testable import TuistGenerator
+
+extension WorkspaceDescriptor {
```

Comment:
What about moving this to `TuistGeneratorTesting` and renaming the file to `WorkspaceDescriptor+TestData.swift`?